sources/Launcher/DYNRobustnessAnalysisLauncher.h

```diff
@@ -181,6 +181,14 @@ class RobustnessAnalysisLauncher {
    * @return the multiple jobs instance create thanks to input data
    */
   boost::shared_ptr<multipleJobs::MultipleJobs> readInputData(const std::string& fileName);
+
+  /**
+   * unzip input file in working directory and obtain name of multiple jobs definition file
+   * @param inputFileFullPath full path of the zip input file
+   *
+   * @return the full path of the unzipped file containing the multiple jobs definition
+   */
+  std::string unzipAndGetMultipleJobsFileName(const std::string& inputFileFullPath);
```

Comment:
This method can be const if I'm not mistaken

Reply:
yes, it does not change the robustness analysis launcher state.